errors.go

```diff
@@ -16,15 +16,17 @@ var (
 	ErrFieldTypeConflict = errors.New("field type conflict")
 )
 
+// ErrDatabaseNotFound indicates that a database operation failed on the specified database.
```

Comment:
You could tweak this with the addition "...because the specified database does not exist".